sa/model.go

```diff
@@ -88,14 +88,18 @@ func ClearEmail(ctx context.Context, dbMap db.DatabaseMap, regID int64, email st
 			return nil, nil
 		}
 
-		currPb.Contact = newContacts
-		newModel, err := registrationPbToModel(currPb)
-		if err != nil {
-			return nil, err
-		}
+		// UPDATE the row with a direct database query, in order to avoid LockCol issues.
+		//
+		// TODO(#7716): Revert to use tx.Update() once LockCol has been removed.
```

Comment:
Honestly I wouldn't leave this TODO here -- it's unlikely that we're going to go back to using `.Update()`, since we're instead headed in the direction of having three separate methods (one for updating contacts, one for updating status, and one for updating JWK), and each of them will just `UPDATE` a single column.